src/boost_histogram/_internal/hist.py

```diff
@@ -597,8 +597,12 @@ def _axis(self, i: int = 0) -> Axis:
     def storage_type(self) -> Type[Storage]:
         return cast(self, self._hist._storage_type, Storage)  # type: ignore[return-value]
```

Comment:
One more thing, unrelated this PR: shouldn't this be a property instead of a function?

Reply:
It is (you have to expand the GitHub view to see the `@property`). However, `_storage_type` also needs to be a property.